sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/models/EncryptionDatabaseResponse.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.EncryptionCosmosAsyncClient;
+
+public class EncryptionDatabaseResponse {
+
+    private final CosmosDatabaseResponse databaseResponse;
+
+    private final EncryptionCosmosAsyncClient encryptionCosmosClient;
+
+    public EncryptionDatabaseResponse(
+        CosmosDatabaseResponse databaseResponse,
+        EncryptionCosmosAsyncClient encryptionCosmosClient) {
+        this.databaseResponse = databaseResponse;
+        this.encryptionCosmosClient = encryptionCosmosClient;
+    }
+
+    public CosmosDatabaseResponse getDatabaseResponse() {
+        return databaseResponse;
+    }
+
+    public EncryptionCosmosAsyncClient getEncryptionCosmosClient() {
```

Comment:
this public api ties the response to the notion of async-ness.

the response should be agnostics of async-ness vs sync-ness